portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,82 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"><script src="script.js"></script>
```

Comment:
nit:
JS Best practices: Place scripts after closing the body tag.

https://faqs.skillcrush.com/article/176-where-should-js-script-tags-be-linked-in-html-documents